apps/email/index.html

```diff
@@ -32,6 +32,7 @@
   <link rel="stylesheet" type="text/css" href="shared/style/progress_activity.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/tabs.css" >
   <link rel="stylesheet" type="text/css" href="shared/style/value_selector.css" >
+  <link rel="stylesheet" type="text/css" href="shared/elements/gaia-icons/gaia-icons.css" >
```

Comment:
> However, if you know for sure that carets are explicitly out for the LTR and RTL cases

They should be there and mirrored. This is a regression coming from one of our Phoxygen patch earlier in the RTL refacto process (the one you've linked in this bug). I didn't see this at first. Sorry for that!

So I'm going for 1/ and will make a patch in bug 1218547 to fix it.